tests/unit/amo/components/TestAddonCompatibilityError.js

```diff
@@ -42,11 +47,10 @@ describe(__filename, () => {
   }
 
   it('renders a notice for non-Firefox browsers', () => {
-    const root = render({
-      lang: 'en-GB',
-      reason: INCOMPATIBLE_NOT_FIREFOX,
-      userAgentInfo: { browser: { name: 'Chrome' }, os: {} },
+    _dispatchClientMetadata({
```

Comment:
Not sure why this was made when we could use the existing `dispatchClientMetadata` helper.

Reply:
With this helper I don't have to pass `store` into the function every time